packages/jest-util/src/invariant.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default function invariant(
+  condition: unknown,
+  message?: string,
```

Comment:
It would be nice to make the message required. Also it would be nice to append "This is a bug in Jest...", like here:

https://github.com/jestjs/jest/blob/0fd5b1c37555f485c56a6ad2d6b010a72204f9f6/packages/jest-runtime/src/index.ts#L530

https://github.com/jestjs/jest/blob/0fd5b1c37555f485c56a6ad2d6b010a72204f9f6/packages/jest-runtime/src/index.ts#L546

`invariant()` is used in places there value is defined, but TypeScript cannot see that. So a failure is definitely a bug, that is why it makes sense to append "This is a bug in Jest..."